### PR TITLE
Flutter iOS support

### PR DESCRIPTION
**Pull request by zhileichen**
This PR intends to add iOS support for flutter

### CODE REVIEW

**Review by unsuitable001 — APPROVED** [review]
Thank you so much @zhileichen for this valuable PR. LGTM.

Related #5 

cc @dcharkes

**Comment by dcharkes** [comment]
@unsuitable001 It looks like the formatter isn't happy on the CI. Also on files which aren't touched by this PR. Could you take a look to get the CI happy before merging this PR?

**Review by dcharkes — APPROVED** [review]
lgtm when CI is green.

Thanks @zhileichen !

File: lib/src/constants.dart
Comment (unsuitable001) [inline lib/src/constants.dart:47]: Change it to
```dart
  } else if (!(Platform.isLinux ||
      platform.startsWith('linux') ||
```

**Review by unsuitable001 — CHANGES_REQUESTED** [review]
Also, in https://github.com/zhileichen/cronet.dart/blob/baeab391e8c3059bd4a24bcf8cc9d5e89d782a1e/lib/src/exceptions.dart#L21

Change it to:

```dart
    final b = StringBuffer()
      ..write('HttpException: ')
      ..write(message);
```

These 2 changes should be enough to make the analyser happy :)

**Comment by dcharkes** [comment]
I just noticed that we didn't bump the version to 0.0.7 and added no entry in the changelog.md Without a version bump I cannot publish a new version of the package on pub.dev. Would you mind making a PR to bump the version and update the changelog?

**Pull request merged**
